docs/introduction/3-component-lifecycle.mdx

```diff
@@ -0,0 +1,59 @@
+import { Meta, Intro, Status } from '../../.storybook/components';
+
+<Meta title="Introduction/Component Lifecycle" />
+
+# Component Lifecycle
+
+<Intro>
+  Circuit UI components move through different stages throughout their
+  lifecycle. Within each stage, components meet different requirements and
+  receive different levels of support.
+</Intro>
+
+## Stable
+
+<Status variant="stable" aria-hidden />
+
+Stable components solve a proven use case. They meet all of our quality standards including accessibility, [browser support](Introduction/Browser-Support/Docs), test coverage, and documentation. Changes to the component API are unlikely and follow [semantic versioning](https://semver.org/).
+
+## Under Review
+
+<Status variant="in-review" aria-hidden />
+
+Our quality requirements evolve over time. Existing components that haven't been updated yet to meet the latest standards are placed under review. You can continue to use them but be aware of the issues that are listed in the components' documentation.
```

Comment:
Nice! So when we look at components in review, let's make sure to document why they're not stable